doc/user_requirements.md

```diff
@@ -0,0 +1,236 @@
+# System Requirement Specification (SRS) Virtual Schema Common Java (VSCJ) 
+
+This document contains the system requirements for the Java bottom-layer base library for all Exasol Virtual Schema adapters.
+
+## Features
+
+### Property Validation
+`feat~property-validation~1`
+
+The VSCJ library provides the infrastructure for validating the input coming from virtual schema properties.
```

Comment:
Can you add a link providing details about what `virtual schema properties` are?

Reply:
Ah, I see you have details below.

Reply:
Added terms section and referenced it.